gef-ui/server/index.js

```diff
@@ -29,6 +29,17 @@ const cookie = {
   maxAge: 604800000, // 7 days
 };
 
+/**
+ * `secure` cookie for HTTPS environments, whilst `false` for localhost.
+ */
+app.use((req, res, next) => {
+  const { host } = req.headers;
+  if (host) {
+    cookie.secure = Boolean(host.indexOf('localhost'));
```

Comment:
I know we discussed this earlier. I wonder though if we can use an environment variable instead? by default it would not be set. but if it's in Azure, then we set it to `true`. what do  you think?

Reply:
I looked into `env` variable, do I need to add that env variable in the docker-compose?

Reply:
not necessary. because we can leave it out by default. and only add it in Azure.
we can say:
if it exists, then set the cookie to `secure` mode.
If it doesn't, then don't do anything - which means we can leave it out of docker compose

Reply:
Okay cool I'll update

Reply:
Updated to `HTTPS` environment variable.